docs/en/Tiered-Locality.md

```diff
@@ -35,7 +35,9 @@ alluxio.locality.[tiername]=...
 See the [Configuration-Settings](Configuration-Settings.html) page for details on how
 to set configuration properties.
 
-It is also possible to configure tiered identity info via script. To use a script, set
+It is also possible to configure tiered identity info via script. By default Alluxio looks
+for a script at `${ALLUXIO_HOME}/conf/tiered_identity.sh`. You can override this location
```

Comment:
what happens if the script path does not exist, or if the script does not output the correct info?

Reply:
if the script path doesn't exist, it will be ignored. This way, we can set a default path for the script and it's ok for the user not to use it. If the script outputs incorrect info, an exception will be thrown.

Reply:
I think this type of information should be in the doc? Is it already there?

Reply:
done